tests/test_warc_metadata_sidecar.py

```diff
@@ -150,9 +151,11 @@ def test_metadata_sidecar_dns_record(self, mock_warcwriter, caplog, tmpdir):
     @patch('warc_metadata_sidecar.find_language')
     @patch('warc_metadata_sidecar.determine_soft404')
     def test_metadata_sidecar_image_record(self, mock_404, mock_language, mock_character, tmpdir):
+        mime_dict = json.dumps({'fido': 'image/gif', 'python-magic': 'image/gif'})
+        puid = json.dumps('fmt/4')
```

Comment:
Could do `mime_dict = '{"fido": "image/gif", "python-magic": "image/gif"}'`.
We should be able to forego the `json.dumps` on the 'fmt/4' once we change the script.